src/bindings/js/node/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+project(ov_node_addon)
+
+set(CMAKE_CXX_STANDARD 17)
+add_definitions(-DNAPI_VERSION=6)
+
+include(FetchContent)
+
+FetchContent_Declare(
+    node-api-headers
+    URL      https://github.com/nodejs/node-api-headers/archive/refs/tags/v1.1.0.tar.gz
+    URL_HASH SHA256=70608bc1e6dddce280285f3462f18a106f687c0720a4b90893e1ecd86e5a8bbf
+)
+FetchContent_MakeAvailable(node-api-headers)
+
+FetchContent_Declare(
+    node-addon-api
+    URL      https://github.com/nodejs/node-addon-api/archive/refs/tags/v7.0.0.tar.gz
+    URL_HASH SHA256=e0cf2ef9bed9965ac8f498b4a7007b9f0002fa022c1ff665f256ab54260d6305
+)
+FetchContent_MakeAvailable(node-addon-api)
+
+add_library(${PROJECT_NAME} SHARED
```

Comment:
Check if static/shared lib should be managed by OpenVINO global flag instead set it here.

Reply:
We require to have shared library version here. There are no options to compile bindings as static lib.